### PR TITLE
README and CHANGELOG for alpha

### PR DESCRIPTION
**Pull request by loganj**
Ready for review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,104 @@
...
-  public final String trackId;
+_"Memory is the treasury and guardian of all things._" - Cicero
 
-  public TrackScreen(String albumId, String trackId) {
-    this.albumId = albumId;
-    this.trackId = trackId;
-  }
-}
-```
+**Flow gives names to your Activity's UI states, navigates between them, and remembers where it's been.**
```
Comment (rjrjr) [inline README.md]: Sweet.

File: README.md [hunk 2]
```diff
@@ -1,67 +1,104 @@
...
-    <groupId>com.squareup.flow</groupId>
-    <artifactId>flow</artifactId>
-    <version>(insert latest version)</version>
-</dependency>
-```
-or Gradle:
-```groovy
-compile 'com.squareup.flow:flow:(latest version)'
+  public boolean equals(Object o) {
+    return (o != null) &&
```
Comment (rjrjr) [inline README.md]: You don't need to null check before instanceof

File: README.md [hunk 3]
```diff
@@ -1,67 +1,123 @@
...
+
+`Flow#set(key)` -- Goes to the requested key. Goes back or forward depending on whether the key is already in the History.
+
+Flow also lets you rewrite history safely and easily.
+
+`Flow#setHistory(history, direction)` -- Change history to whatever you want.
+
+See the [Flow][Flow.java] class for other convenient operators.
 
+### History
```
Comment (ChrisRenke) [inline README.md]: More philosophical note: This section is called "History" but seems primarily to be about view state; those two things are decoupled in implementation, perhaps they should be two sections here? Or rename this "Maintaining State" and let the previous "Navigation" section handle the "As you navigate the app, Flow keeps track of where you've been"

Comment (loganj) [inline README.md]: Maybe just merge it into "Navigation", and retitle that section "Navigation and History"?

**Pull request merged**